composer.json

```diff
@@ -44,7 +44,8 @@
         "symfony/phpunit-bridge": "^5.2",
         "symfony/templating": "^3.4|^4.3|^5.0",
         "symfony/validator": "^3.4|^4.3|^5.0",
-        "symfony/yaml": "^3.4|^4.3|^5.0"
+        "symfony/yaml": "^3.4|^4.3|^5.0",
+        "friendsofphp/php-cs-fixer": "*"
```

Comment:
You have to remove this line as we don't use PHP-CS-Fixer as a dev dependency anymore.

Reply:
But I don't have global php-cs-fixer and the only way to use it is as dependency. That way package is fully packaged with all dependencies. Isn't it better?

Reply:
Why don't you use `composer global require friendsofphp/php-cs-fixer`, export PATH="$PATH:$HOME/.composer/vendor/bin" and `php-cs-fixer`?